Makefile

```diff
@@ -131,215 +131,12 @@ debug-flatpak:
 uninstall-flatpak:
 	$(FLATPAK) uninstall org.nanuc.Axolotl
 
-## Snap
+clean-flatpak:
+	rm -rf flatpak/.flatpak-builder flatpak/build
+
+# Snap
 build-snap:
 	@sudo $(SNAPCRAFT)
 
 install-snap:
 	@sudo $(SNAP) install axolotl_$(AXOLOTL_VERSION)_amd64.snap --dangerous
-
-## Debian arm64 building/cross-compiling and packaging on Debian 'testing'
-## Please install the packages git and build-essential before getting the source via
-## 'git clone --depth=1 https://github.com/nanu-c/axolotl/'
-## and run 'make dependencies-deb-arm64(-cc)' once.
-
-check-platform-deb-arm64:
-ifneq ($(UNAME_S),Linux)
-	@echo "Platform unsupported - only available for Linux" && exit 1
-endif
-ifneq ($(HARDWARE_PLATFORM),aarch64)
-	@echo "Machine unsupported - only available for arm64/aarch64" && exit 1
-endif
-ifneq ($(APT),/usr/bin/apt)
-	@echo "OS unsupported - apt not found" && exit 1
-endif
-ifneq ($(DEBIAN_VERSION),bookworm)
-	@echo "Debian version not support - 'testing' is needed" && exit 1
-endif
-
-dependencies-deb-arm64: check-platform-deb-arm64
-	@echo "Installing dependencies for building Axolotl on Debian 'testing' (bookworm)..."
-	@sudo $(APT) update
-	@sudo $(APT) install --assume-yes curl wget nodejs yarn debmake
-	@sudo $(APT) install --assume-yes --no-install-recommends libgtk-3-dev libjavascriptcoregtk-4.1-dev libsoup-3.0-dev libwebkit2gtk-4.1-dev protobuf-compiler
-ifneq ($(RUST),${HOME}/.cargo/bin/rustup)
-	@curl --proto '=https' --tlsv1.2 -sSf https://sh.rustup.rs | sh -s -- -y
-endif
-	@$(CARGO_PREFIX)/rustup update
-	@echo "Dependencies installed."
-
-build-deb-arm64: clean-deb-arm64
-	@echo "Building Axolotl for arm64/aarch64 on Debian - Please use 'testing' release!)."
-	@echo "Installing dependencies (yarn)..."
-	@cd $(CURRENT_DIR)/axolotl-web && yarn install --frozen-lockfile
-	@echo "Building (yarn)..."
-	@cd $(CURRENT_DIR)/axolotl-web && yarn run build
-	@echo "Building (rust)..."
-	$(CARGO_PREFIX)/cargo build --features tauri --release
-	@echo "Building complete."
-
```

Comment:
How is the deb package now built? `cargo tauri build --ci --target aarch64-unknown-linux-gnu --features tauri --bundles deb` -> does it work?

Reply:
`--ci` is only needed for non-interactive builds.
`--target aarch64-unknown-linux-gnu` is only needed for cross-compilation.

So if you want to build for your current arch, it is only `cargo tauri build --features tauri --bundles deb`.